src/main/java/gtclassic/fluid/GTFluid.java

```diff
@@ -17,7 +17,16 @@
 	protected static SoundEvent emptySound = SoundEvents.ITEM_BUCKET_EMPTY;
 	protected static SoundEvent fillSound = SoundEvents.ITEM_BUCKET_FILL;
 	protected static Material material = Material.WATER;
-	GTMaterial mat = null;
+	GTMaterial mat;
+	
+	private GTFluid(GTMaterial mat, String base, String suffix, boolean isGaseous) {
+		super(mat.getDisplayName().toLowerCase() + suffix, new ResourceLocation(GTMod.MODID, "fluids/"
+				+ base), new ResourceLocation(GTMod.MODID, "fluids/flowing"));
+		this.mat = mat;
+		this.temperature = 300;
+		this.mapColor = mat.getColor().hashCode();
```

Comment:
getRGB. not hashcode. Even if they return the same thing. Its like using a comparator for a Object2ObjectFunction.

Reply:
added in #86